modules/apps/portal-language/portal-language-override-test/src/testFunctional/tests/LanguageOverrideExportImport.testcase

```diff
@@ -42,4 +42,25 @@ definition {
 		}
 	}
 
+	@description = "This test asserts that the File Upload field is required in Import Translations page."
+	@priority = "4"
+	test CanViewFileUploadFieldIsRequired {
+		task ("Given go to the Import Translations page") {
+			Portlet.gotoPortletOptions(portletOption = "Import Translations");
+		}
+
+		task ("When click Save button") {
+			Button.clickSave();
+		}
+
+		task ("Then assert error message and File Upload field is required.") {
+			FormPortlet.viewValidationErrorMessage();
+
+			TestUtils.assertPartialAttributeValue(
```

Comment:
Good job for finding the macro, there is a more readable macro in _FormFields.macro_, could you please use that one instead? Thanks in advance.

Reply:
Use FormFields.viewFieldRequired() and update the path value of FormFields#FIELD_REQUIRED